libp2p/crypto/curve25519.nim

```diff
@@ -97,3 +100,5 @@ proc mulgen*(_: type[Curve25519], dst: var Curve25519Key, point: Curve25519Key)
 proc public*(private: Curve25519Key): Curve25519Key =
   Curve25519.mulgen(result, private)
 
+proc random*(_: type[Curve25519Key]): Curve25519Key =
+  discard randomBytes(result)
```

Comment:
You MUST not discard result value of `randomBytes()` and return an error if it returns not enough bytes.

Reply:
Fixed, thanks